salt/metalk8s/archives/unmounted.sls

```diff
@@ -0,0 +1,10 @@
+{% set removable_archives = salt.metalk8s.get_removable_archives() %}
+{% for env, archive in removable_archives.items() %}
+{% if archive.iso %}
+Archive {{ archive.iso }} absent at {{ archive.path }}:
```

Comment:
I know it comes from the state name from `mounted.sls` but I suggest putting a name a bit clearer
```suggestion
Unmount archive {{ archive.iso }} from {{ archive.path }}:
```